com.unity.cinemachine/Documentation~/CinemachineInputAxisController.md

```diff
@@ -28,3 +31,142 @@ If you like, you can also use CinemachineInputAxisController with your own scrip
+public class SliderControllerEditor : Unity.Cinemachine.Editor.InputAxisControllerEditor {}
+#endif
+```
+
+## Using PlayerInput and PlayerInputManager components with Cinemachine
+
+For more complex input configuration like supporting multiple devices, local multiplayer... You will need to receive inputs from the `PlayerInput` component provided by the Input System package. This section is assuming you already know how to setup this component. All the documentation can be found on the Input System package documentation page and samples.
+
+### Read from PlayerInput component
+
+To read values from a `PlayerInput` with a `behaviour` set to `InvokeCSharpEvents`, You need to create a custom `InputAxisController` that subscribes to `onActionTriggered`. The example bellow shows how to receive and wire those inputs accordingly. Add this script to your `CinemachineCamera` and assign the `PlayerInput` field.
+
+```cs
+using UnityEngine;
+using Unity.Cinemachine;
+using System;
+using UnityEngine.InputSystem;
+
+class PlayerInputReceiver : InputAxisControllerBase<SimpleReader>
+{
+    [SerializeField]
+    PlayerInput m_PlayerInput;
+
+    void Awake()
+    {
+        // Call back when the PlayerInput receives an Input.
+        m_PlayerInput.onActionTriggered += OnActionTriggered;
+    }
+
+    public void OnActionTriggered(InputAction.CallbackContext value)
+    {
+        // Sends the Input to all the controllers.
+        for (var i = 0; i < Controllers.Count; i ++)
+        {
+            Controllers[i].Input.SetValue(value.action);
+        }
+    }
+}
+
+[Serializable]
+class SimpleReader : IInputAxisReader
+{
+    // Assumes the action is a Vector2 for simplicity but can be changed for a float.
+    Vector2 m_Value;
+    [SerializeField]
+    InputActionReference m_Input;
+    [SerializeField]
+    float m_Gain;
+    
+    public void SetValue(InputAction action)
+    {
+        // Is the input referenced in the inspector matches the updated one update the value.
+        if (m_Input!= null && m_Input.action.id == action.id)
+            m_Value = action.ReadValue<Vector2>();
+    }
+
+    public float GetValue(UnityEngine.Object context, 
+        int playerIndex, 
+        bool autoEnableInput, 
+        IInputAxisOwner.AxisDescriptor.Hints hint)
+    {
+        if(hint == IInputAxisOwner.AxisDescriptor.Hints.X)
+            return m_Value.x * m_Gain;
+        
+        return m_Value.y * m_Gain;
+    }
+}
+
+#if UNITY_EDITOR
+[UnityEditor.CustomEditor(typeof(PlayerInputReceiver))]
+public class PlayerInputReceiverEditor : Unity.Cinemachine.Editor.InputAxisControllerEditor {}
+#endif
+```
+
+### Local split screen multiplayer
+
+In a split screen multiplayer environment using `PlayerInputManager` and `PlayerInput` components you will need one `CinemachineBrain` and one `CinemachineCamera` per player `Camera` and increase the channel mask of both the `CinemachineCamera` and the `CinemachineBrain`.
```

Comment:
This is still pretty thin and hard to understand.  I'm going to leave it to @JWalker0007 to help you express this more effectively.

Reply:
Moved it in the CinemachineBrain.md and referenced it instead. It's a script that could be used in this case but has nothing to do with the Inputs. Maybe that's where the confusion started.
Let me know if it's more clear.

Reply:
There is already a page for SplitScreen and Multiple Unity Cameras.  I think it would be better there instead of in CinemachineBrain.